linera-core/src/local_node.rs

```diff
@@ -75,6 +76,11 @@ pub enum LocalNodeError {
 
     #[error("Got different blob id from downloaded blob when trying to download {blob_id:?}")]
     BlobIdMismatch { blob_id: BlobId },
+
+    #[error(
+        "Got different hash from downloaded certificate value when trying to download {hash:?}"
+    )]
+    CertificateValueHashMismatch { hash: CryptoHash },
```

Comment:
If they are different, maybe one could put both values in the error?
That could help the debugging.